extensions/functions/src/test/java/io/crate/window/OffsetValueFunctionsTest.java

```diff
@@ -559,4 +568,42 @@ public void test_offset_following_supports_interval_in_range_mode_simple() throw
             INTERVAL_DATA
         );
     }
+
+    @Test
+    public void test_row_cache_memory_is_accounted() throws Throwable {
+        RamAccounting accounting = new RamAccounting() {
```

Comment:
Maybe worth adding a `PlainRamAccounting` with a max parameter to the test fixtures to replace the almost identical definitions in `SketchRamAccountingTest` and `PercentileAggregationTest`

Reply:
Done in https://github.com/crate/crate/pull/16054/commits/37099f5bd5d57909843d19b6aece7cb2c4330fef